src/modules/auth/wrappers/with-auth/with-auth.tsx

```diff
@@ -60,10 +59,11 @@ export const withAuth = (
 			checkLoginStatus();
 		}, [checkLoginStatus]);
 
-		if (!isLoggedIn) {
-			return <Loading fullscreen />;
-		}
-
-		return <WrappedComponent {...props} />;
+		return (
+			<>
+				{!isLoggedIn && <Loading fullscreen />}
+				<WrappedComponent {...props} />
```

Comment:
doesn't this risk loading the spinner and the component at the same time?

Reply:
Intentional; this prevents white-flash-of-content when the logged in status re-checks.